influxdb_client/client/write/retry.py

```diff
@@ -58,18 +76,26 @@ def get_backoff_time(self):
         if consecutive_errors_len < 0:
             return 0
 
-        backoff_value = self.backoff_factor * (self.exponential_base ** consecutive_errors_len) + self._jitter_delay()
-        return min(self.max_retry_delay, backoff_value)
-
-    def get_retry_after(self, response):
-        """Get the value of Retry-After header and append random jitter delay."""
-        retry_after = super().get_retry_after(response)
-        if retry_after:
-            retry_after += self._jitter_delay()
-        return retry_after
+        delay_range = self.backoff_factor
+        i = 1
+        while i <= consecutive_errors_len:
+            i += 1
+            delay_range = delay_range * self.exponential_base
+            if delay_range > self.max_retry_delay:
```

Comment:
@alespour had a good point that the delays should be increasing (on average), this condition makes it hard to happen since the delay range is the same after a fixed count of attempts, the delays are then oscillating randomly around `self.max_retry_delay/2`. This can be fixed by restricting the delay range to a large number herein:
```suggestion
            if delay_range > 100_000_000:
```